TESTS/mbed-crypto/sanity/main.cpp

```diff
@@ -438,7 +438,9 @@ utest::v1::status_t case_teardown_handler(const Case *const source, const size_t
 
 utest::v1::status_t test_setup(const size_t number_of_cases)
 {
+#ifndef NO_GREENTEA
```

Comment:
This is dependency, there's another PR adding this to test framework?

Reply:
Then should we drop this commit?